modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/ComposerServiceImpl.java

```diff
@@ -918,6 +933,22 @@ public Job concat(String profileId, Dimension outputDimension, boolean sameCodec
     return concat(profileId, outputDimension, -1.0f, sameCodec, tracks);
   }
 
+  @Override
+  public Job mergeAudioTracks(String profileId, List<Long> audioStartTimes, List<Track> audioTracks) throws MediaPackageException, EncoderException {
+    try {
+      ArrayList<String> arguments = new ArrayList<String>();
+      arguments.add(0, profileId);
+      String audioStartsStringified = audioStartTimes.stream().map(String::valueOf).collect(Collectors.joining(","));
+      arguments.add(1, audioStartsStringified);
+      for (int i = 0; i < audioTracks.size(); i++) {
```

Comment:
I'm not a fan of this approach, but `getAsXml` throws exceptions so we can't turn this into a functional call :(